data/json/recipes/armor/bespoke_armor/survivor.json

```diff
@@ -9,48 +9,35 @@
     "difficulty": 6,
     "time": "10 h",
     "autolearn": true,
-    "using": [ [ "sewing_kevlar", 200 ], [ "fabric_lycra", 20 ], [ "fabric_fauxfur", 20 ] ],
+    "using": [ [ "sewing_kevlar", 200 ], [ "fabric_lycra", 20 ], [ "fabric_fauxfur", 20 ], [ "fastener_large", 1 ], [ "clasps", 8 ] ],
     "qualities": [ { "id": "LEATHER_AWL", "level": 1 } ],
     "proficiencies": [
       { "proficiency": "prof_closures" },
       { "proficiency": "prof_closures_waterproofing" },
       { "proficiency": "prof_polymerworking" },
       { "proficiency": "prof_elastics" }
     ],
-    "components": [
-      [ [ "coat_rain", 1 ], [ "jacket_windbreaker", 1 ], [ "jacket_evac", 1 ], [ "coat_gut", 1 ] ],
-      [ [ "duct_tape", 300 ] ],
-      [ [ "sheet_kevlar_layered", 24 ] ]
-    ]
+    "components": [ [ [ "sheet_kevlar", 130 ] ] ]
```

Comment:
These seem really low as values. A layered Kevlar panel is 16 sheets, so these counts should be close to 16 * 24 (in this case). So at least double the cost of Kevlar, realistically close to 3 times.

Reply:
Part of why I did the PR is that after the tailoring audit the materials all weighed more. A Layered Kevlar Panel is, after the tailoring audit, 429g so 24 of them would be 10.296kg. This is more than the suit weighs by a large factor. the entire suit including all materials weighs only 8.4kg. the Faux Fur sheets themselves weigh in at 372g apiece for 7.44kg. The total is more than twice the weight of the final product and while there should be some wastage 50% is too much. Either I have to reduce the materials or increase the weight of the final product.

I can do either, BTW, but I chose to reduce the mats.

   - Shane